sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/NamespacePropertiesExtensions.cs

```diff
@@ -3,34 +3,40 @@
 
 using System;
 using System.Globalization;
+using System.Threading.Tasks;
 using System.Xml.Linq;
+using Azure.Core.Pipeline;
 
 namespace Azure.Messaging.ServiceBus.Administration
 {
     internal class NamespacePropertiesExtensions
     {
-        public static NamespaceProperties ParseFromContent(string xml)
+        public static async Task<NamespaceProperties> ParseFromContentAsync(Response response, ClientDiagnostics diagnostics)
```

Comment:
Should this still be `ParseFromContent` as it now takes a Response instead of content XML? `ParseFromResponse`? (This would apply to all versions of this method from all the extension classes)